tests/cases/php/empty-statement

```diff
@@ -0,0 +1,28 @@
+---DESCRIPTION---
+Statements must not be empty, except catch blocks
+---CONTENTS---
+<?php
+
+declare(strict_types=1);
+
+if ($foo) {
+    // Bar
```

Comment:
why comments do not count? `// a long explanation of why we shouldn't do anything here` should suffice?

Reply:
Can we find a valid example of this? Will take a look.

Reply:
Could find anything locally. Searched Symfony as it's a large codebase, there are a few, such as:

- https://github.com/symfony/symfony/blob/737726516e819fc0b1ea43dc6491bddd6d98b1f4/src/Symfony/Component/DependencyInjection/Container.php#L161-L163
- https://github.com/symfony/symfony/blob/737726516e819fc0b1ea43dc6491bddd6d98b1f4/src/Symfony/Component/Debug/ExceptionHandler.php#L123-L125
- https://github.com/symfony/symfony/blob/737726516e819fc0b1ea43dc6491bddd6d98b1f4/src/Symfony/Component/PropertyAccess/PropertyAccessor.php#L339-L341
- https://github.com/symfony/symfony/blob/737726516e819fc0b1ea43dc6491bddd6d98b1f4/src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler.php#L120-L122

I think this cases are all avoidable (by returning early etc), but maybe this should just be something for code review.

Reply:
These examples are what I had in mind, they are edge cases of complex code

Reply:
Going to ignore for now. If there really is a situation where it makes sense, then we can revisit/disable that individual check.